chart/kubeapps/Chart.yaml

```
@@ -33,4 +33,4 @@ maintainers:
 name: kubeapps
 sources:
   - https://github.com/kubeapps/kubeapps
-version: 7.5.11-dev0
+version: 7.5.11-dev1
```

Small update after chart changes resynced.
```suggestion
version: 7.5.12-dev1
```